src/storage/mod.rs

```diff
@@ -1357,6 +1420,12 @@ mod tests {
             )
             .unwrap();
         rx.recv().unwrap();
+        expect_batch_get_vals(
+            vec![None],
```

Comment:
Why only one `None` here?

Reply:
Errors and not found values behaves different. Before the read pool, it is already single None here.

Reply:
OK, I get it, this `None` means I get an error when getting "c", and since "d" is not found, I get nothing, so just one `None` here.